include/seastar/http/request.hh

```diff
@@ -76,7 +76,8 @@ struct request {
     std::unordered_map<sstring, sstring, case_insensitive_hash, case_insensitive_cmp> _headers;
     std::unordered_map<sstring, sstring> query_parameters;
     httpd::parameters param;
-    sstring content; // server-side deprecated: use content_stream instead
+    temporary_buffer<char> content; // server-side deprecated: use content_stream instead
```

Comment:
Without [[deprecated]], we don't know if it's safe to reuse.

Also, it doesn't work for large contents. Can't we use content_stream, but with known-in-advance size?

And wrt S3/Scylla, how will we know the size in advance?

Reply:
> Can't we use content_stream, but with known-in-advance size?

Yes, but it's not as nice

> And wrt S3/Scylla, how will we know the size in advance?

S3 _forces_ clients to know size in advance. It doesn't accept chunked encoding even for partial-upload finalize message :(

Reply:
Let's deprecate content for both server and client side, with a [[deprecated]] signal.

Reply:
Can't we override them with #pragma?